lib/proofer/vendor/mock.rb

```diff
@@ -109,6 +112,14 @@ def build_question_set(_vendor_resp)
 
       private
 
+      def financial_errors(financials)
+        errors = {}
+        financials.keys.each do |key|
+          errors[key] = "The #{key} could not be verified."
+        end
+        errors
```

Comment:
Actually, it might be something that Rubocop flags. We should enable Code Climate for this repo.